src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
```

Comment:
Please proofread [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)
You should create separate file for each BEM block